src/rgw/rgw_rados.h

```diff
@@ -254,6 +254,14 @@ class RGWObjManifest {
                              as object might have been copied across buckets */
   map<uint64_t, RGWObjManifestRule> rules;
 
+  /* If this object is copied from another versioned object, @src_instance 
+   * is used to store src's version_id. 
+   * 
+   * @copied_obj is used to indicate whether @src_instance is valid.
+   * */
+  string src_instance; 
+  bool copied_obj;
```

Comment:
@yanghonggang do we really need to keep copied_obj field? Isn't it that we should just check to see that `!src_instance.empty()`?